docs/podman.1.md

```diff
@@ -198,15 +200,15 @@ the exit codes follow the `chroot` standard, see below:
 
     registries.conf is the configuration file which specifies which container registries should be consulted when completing image names which do not include a registry or domain portion.
 
-    When Podman runs in rootless mode, the file `$HOME/.config/containers/registries.conf` is used.
+    When Podman runs in rootless mode, the file `$HOME/.config/containers/registries.conf` replaces the system default, if it exists.
```

Comment:
I would not use the word "replaces"; it has an active implication suggesting actual overwriting. Possibly just say it is "used" or "consulted" instead of the system default.

Reply:
Fixed with
`    Non root users of Podman can create the `$HOME/.config/containers/registries.conf` file to be used instead of the system defaults.`